msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AcquireTokenSilentSupplier.java

```diff
@@ -13,6 +13,8 @@
 class AcquireTokenSilentSupplier extends AuthenticationResultSupplier {
 
     private SilentRequest silentRequest;
+    private boolean shouldRefresh;
```

Comment:
This seems to be a local variable in the "execute" method, no need to have this as obj state.

Reply:
Fixed in latest commit.